ted_data_eu/services/etl_pipelines/ted_data_etl_pipeline.py

```diff
@@ -317,7 +335,7 @@ def generate_notice_link(lot_url):
 
         # change field codes with labels
         data_table[LOT_NUTS_0] = data_table[LOT_NUTS_0].apply(
-            lambda x: pycountry.countries.get(alpha_2=x).name if x else None)
+            lambda x: get_country_name_by_code(x) if x else None)
```

Comment:
remove if x else None, function get_country_name_by_code already check if is None